sw/host/tests/manuf/provisioning/ft/src/main.rs

```diff
@@ -13,11 +13,21 @@ use opentitanlib::backend;
 use opentitanlib::dif::lc_ctrl::DifLcCtrlState;
 use opentitanlib::test_utils::init::InitializeTest;
 use opentitanlib::test_utils::load_sram_program::SramProgramParams;
+use ujson_lib::provisioning_data::ManufFtIndividualizeData;
 use util_lib::hex_string_to_u32_arrayvec;
 
 /// Provisioning data command-line parameters.
 #[derive(Debug, Args, Clone)]
 pub struct ManufFtProvisioningDataInput {
+    /// Device ID to provision.
+    ///
+    /// Must match the device ID provisioned in to flash during CP, if one was provisioned then.
+    #[arg(
+        long,
+        default_value = "0xCAFEBABE_11112222_33334444_55556666_77778888_9999AAAA_BBBBCCCC_DEADBEEF"
+    )]
```

Comment:
I'm a little concerned about providing a default value for this at all - do we run the risk of creating several devices with identical IDs because of forgetting to say `--device_id=<foo>` on the command line?

I _think_ if you do not supply a `default_value` and your struct doesn't implement `Default`, then clap will make this a required argument and you must specify it on the command line.

Reply:
Yes, I agree. I just did this temporarily for testing purposes. I plan to refactor this in a follow up where I will move the provisioning binaries to be just driven by `rust_binary`s that have requried cmd line args. This is being tracked in this is currently tracked via #20212.